src/index.js

```diff
@@ -22,15 +22,17 @@ export default function (options = {}, callback = function noop () {}) {
     if (status === 0) {
       callback(status)
     } else {
+      let failedSpecs = parser.parse(output)
       if (++testAttempt <= parsedOptions.maxAttempts) {
         log('info', `Using ${parser.name} to parse output\n`)
```

Comment:
Can we move the log line above the call to `parser.parse` as well? That way we can see what parser we used before we potentially error out